packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -709,7 +709,7 @@ - (FVPTextureMessage *)onPlayerSetup:(FVPVideoPlayer *)player
   [eventChannel setStreamHandler:player];
   player.eventChannel = eventChannel;
   self.playersByTextureId[@(textureId)] = player;
-  FVPTextureMessage *result = [FVPTextureMessage makeWithTextureId:@(textureId)];
+  FVPTextureMessage *result = [FVPTextureMessage makeWithTextureId:textureId];
```

Comment:
is it pigeon's change? the new pigeon supports primitives and the old version does not?

Reply:
Yes, I changed Pigeon's Obj-C generator to stop boxing non-nullable values, to make them more idiomatic and less error-prone (given that it turns out Xcode is less reliable about warning for buggy implicit `NSNumber*`->`BOOL` conversions than I thought).